### DISCUSSION TITLE
Help for Anura3D usage issues

### DISCUSSION THREAD

**Discussion by cw1105-ncist** [Anura3D/Anura3D_OpenSource#135] [Q&A]

**Comment by cw1105-ncist** [comment]


**Reply by FrancescaCeccato** [reply]
What is the quantity of materials that start giving you the differences?
If this number is 30, than you can replace the instruction
MAXNMATERIALS = 30 in GlobalConstants.FOR with a higher number.

**Comment by cw1105-ncist** [comment]


**Reply by FrancescaCeccato** [reply]
Are you using MPM-MIXED or MPM-MP integration?
There will be differences usign MPM-MIXED integration because Gauss integration can be performed only in elements containing MP of the same material, and material-point integration is done in "mixed element" (those with MPs of different materials). This problem does not occur for MPM-MP, which does mpaterial-point integration everywhere.
